Dockerfile

```diff
@@ -43,4 +43,4 @@ WORKDIR $HUGO_HOME
 EXPOSE 1313
 VOLUME  ["$HUGO_HOME"]
 
-ENTRYPOINT ["hugo"]
+CMD ["hugo"]
```

Comment:
I don't like the replacing of the entrypoint see my comment below this review.

Reply:
Ah, okay, didn't know that!
But you can squash all the commits when you merge a PR as a repo manager.
tried this on my branch before but failed glamorously.. Will try again.